tests/spec/core/inlines-spec.js

```diff
@@ -188,6 +188,35 @@ describe("Core - Inlines", () => {
     ]);
   });
 
+  it("proceses `backticks` as code", async () => {
+    const body = `
+      <section>
+        <p id="simple">Return \`null\`.</p>
+        <p id="multi">Return \`123\` or \`undefined\` or \`this
+          particular string\`.
+        </p>
+        <p id="no-match">Return \`\`\`don't match this code blocks\`\`\`</p>
+      </section>
+    `;
+    const doc = await makeRSDoc(makeStandardOps(null, body));
+
+    // simple case
+    const simple = doc.querySelector("#simple code");
+    expect(simple).toBeTruthy();
+    expect(simple.textContent).toBe("null");
+
+    // multi per line
+    const multi = doc.querySelectorAll("#multi code");
+    expect(multi.length).toBe(3);
+    const [firstCode, secondCode, thirdCode] = multi;
+    expect(firstCode.textContent).toBe("123");
+    expect(secondCode.textContent).toBe("undefined");
+    expect(thirdCode.textContent.endsWith("string")).toBeTruthy();
```

Comment:
Wouldn't `multi[i]` be more readable here?

Reply:
yeah, you are right. Will fix.